tests/testthat/test-geom-boxplot.R

```diff
@@ -73,7 +73,7 @@ test_that("boxes with variable widths do not overlap", {
   d <- get_layer_data(p)[c("xmin", "xmax")]
   xid <- find_x_overlaps(d)
 
-  expect_false(any(duplicated(xid)))
+  expect_false(anyDuplicated(xid) > 0)
```

Comment:
I get that `anyDuplicated()` is more efficient, but the mental effort to understand what this expectation is doing is increased by this change.
```suggestion
  expect_equal(anyDuplicated(xid), 0)
```